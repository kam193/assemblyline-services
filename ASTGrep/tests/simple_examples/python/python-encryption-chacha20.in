ChaCha20.new(key=key, nonce=b"123456789123").decrypt(b'os\x9e')
ChaCha20.new(key=b"12345678912345671234567891234567", nonce=b"123456789123").decrypt(b'os\x9e')
