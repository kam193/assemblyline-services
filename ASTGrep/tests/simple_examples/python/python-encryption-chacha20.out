ChaCha20.new(key=key, nonce=b"123456789123").decrypt(b'os\x9e')
b'abc'
